docs/man-pages/condor_hold.rst

```diff
@@ -1,114 +1,122 @@
-      
-
 *condor_hold*
-==============
+=============
+
+Place job(s) into the hold state in HTCondor.
```

Comment:
As per our hallway discussion, perhaps we should try to not use a word in the name of the command (i.e. *hold*) in the first sentence definition of the command, much like the rules for dictionary definitions.  Maybe something like:

"Prevent job(s) from running"